.github/workflows/ci.yml

```diff
@@ -28,9 +28,11 @@ jobs:
       with:
         python-version: "3.x"
     - name: Install dependencies
-      run: pip install cibuildwheel
+      run: pip install cibuildwheel twine
     - name: Build Wheels
-      run: python -m cibuildwheel --output-dir wheelhouse
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
+        twine check wheelhouse/*
```

Comment:
how about running that in a subsequent separate step, separate from the `Build Wheels` one?